examples/documentation/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "examples-documentation"
+version = "0.0.0"
+publish = false
+edition = "2018"
```

Comment:
nit: missing license.

Reply:
I think the license is intended to be used to by crates io to automatically know the license of all packages. We never publish this one so it's not needed.

Reply:
We have licenses for the other `publish = false` crates so maybe we can either add it here as well, or also remove it in those places. My main argument of adding it is that, even if we don't publish this crate, it can still be added as a dependency by the Github repo URL, in which case having a license field may be useful (although, in practice I never see this happening).